### DISCUSSION TITLE
How to configure deploy in AWS Cloudfront?

### DISCUSSION THREAD

**Discussion by ricardojohnny** [chatwoot/chatwoot#1971] [Self hosted] [answered]
Hello,

I still have serious doubts on how to actually use the AWS CDN for Chatwoot.

I was looking at the project documentation at the link below, and I have some questions:

DOC: https://www.chatwoot.com/docs/deployment/cdn/cloudfront

- The whole project, available in an S3 Bucket?
- Do you have a Default Root Object to point to / app?
- Do you have more settings that we have to apply at the CDN, such as "Error Pages"?

I ask this because we are still unable to deploy using the AWS CDN.

**Comment by semsphy** [comment, marked as answer]
1. The whole project, available in an S3 Bucket?
> You store assets on S3 and use cloudfront to handle the assets

2. Do you have a Default Root Object to point to / app?
> Nothing do with Default Root Object because as I mentioned above we host only asset on s3 and serve via cloudfront

3. Do you have more settings that we have to apply at the CDN, such as "Error Pages"?
> No

I deployed chatwoot on AWS Lightsail and my assets store on s3 and serve it via cloudfront.
